app/routes/articles/components/Overview.tsx

```diff
@@ -24,17 +24,17 @@ const OverviewItem = ({ article }: { article: ArticleWithAuthorDetails }) => (
     </h2>
 
     <span className={styles.itemInfo}>
-      {article.author.id && (
-        <span>
-          <Link
-            to={`/users/${article.author.username}`}
-            className={styles.overviewAuthor}
-          >
-            {' '}
-            {article.author.fullName}
-          </Link>{' '}
-        </span>
-      )}
+      {article?.authors?.map((e) => {
```

Comment:
Here as well, `article` should not suddenly be nullish?

Reply:
Realisticly, `authors` is also a required field and should always be defined as well no?